src/global/types.h

```diff
@@ -2036,8 +2034,6 @@ typedef struct SAMPLE_INFO {
     int16_t flags;
 } SAMPLE_INFO;
 
```

Comment:
I'll confirm later today if the game code uses unsafe casts anywhere that night break this.

Reply:
I have checked all casts across the projects and it seems the last unsafe operations were removed with the introduction of XYZ_32 struct. I used this command to look for the casts:

```
grep -P '(?<!sizeof ?)\([^\(\),]*\*\)' **/*.c|grep -v -P '\((const )?(void|u?int(8|16|32)_t|char|bool|float|double) ?(\* ?)?(const ?)?\*\)'
```